cli/src/main.rs

```diff
@@ -237,15 +237,30 @@ fn run() -> Result<(), CliError> {
                     .takes_value(true)
                     .help("Path to private key file"),
             )
+            .arg(
+                Arg::with_name("node_file")
+                    .long("node-file")
+                    .takes_value(true)
+                    .required_unless("node")
+                    .long_help(
+                        "File system path or URL to a .yaml file of node definitions. \
+                         Local files can be loaded from any valid path. \
+                         URLs for remote files must be prefixed with 'http://' or 'https://'. \
+                         A valid .yaml file will be a list nodes, where each node has an \
```

Comment:
If we do keep this description, in a details section, change it to be more example-based.